src/schema/me/create_credit_card_mutation.js

```diff
@@ -34,6 +39,33 @@ export const CreditCardType = new GraphQLObjectType({
   }),
 })
 
+export const CreditCardMutationSuccessType = new GraphQLObjectType({
+  name: "CreditCardMutationSuccess",
+  isTypeOf: data => data.id,
+  fields: () => ({
+    credit_card: {
```

Comment:
Looks like you’re mixing snake and camel case in this PR (e.g. `mutationError` below). Is that intentional?

Reply:
Ah, yeah `credit_card` is the field that's currently used so I can't change that but I'll make sure it's camelCase everywhere else!

Reply:
How do you mean? `CreditCardMutationSuccessType` is a new type and the new fields, right?

Reply:
Oh oh oh you're right. I thought this comment was on a different line 😬